client/my-sites/menus/menu.jsx

```diff
@@ -242,6 +243,7 @@ var Menu = React.createClass( {
 								selectedLocation={ this.props.selectedLocation }
 								setBusy={ this.props.setBusy }
 								confirmDiscard={ this.props.confirmDiscard } />
+						<MenuRevertButton menuData={ this.props.siteMenus } />
```

Comment:
I thought about that and chose "revert" intentionally. When looking at the UI the word "cancel" is clearer about what it does for the user, but I think that "revert" explains more about what the button does at a code level. I decided that it was better for the code to describe what the code does and the UI to describe what the user wants to accomplish, than to have a generalized term for the code that isn't as clear.